zh/core-libraries/xml.rst

```diff
@@ -5,17 +5,155 @@ Xml
 
 .. php:class:: Xml
 
-.. note::
-    The documentation is not currently supported in Chinese language for this
-    page.
+``class Cake\Utility\Xml``
 
-    Please feel free to send us a pull request on
-    `Github <https://github.com/cakephp/docs>`_ or use the **Improve This Doc**
-    button to directly propose your changes.
+Xml类充许将数组转换成 ``SimpleXMLElement`` 或者 ``DOMDocument`` 对象，并且可以转回数组。
 
-    You can referer to the english version in the select top menu to have
-    information about this page's topic.
+创建XML对象
+===========
+
+.. static Cake\Utility\Xml::build($input, array $options =[])
+
+你可使用Xml::build() 载入XML-ish数据。取决于你 ``$options`` 的参数，这个方法允许你返回一个 ``SimpleXMLElement (default) or DOMDocument`` 对象，在各种数据源中，你可使用 ``Xml::build()`` 去创建一个XML对象。例如，你可以通过下面的设置创建XML::
+
+    $text = '<?xml version="1.0" encoding="utf-8"?>
+    <post>
+        <id>1</id>
+        <title>Best post</title>
+        <body> ... </body>
+    </post>';
+    $xml = Xml::build($text);
+    
+你也可以通过本地文件创建Xml对象::
+
+    //本地文件
+    $xml = Xml::build('/home/awesome/unicorns.xml');
+
+你也可以通可一个数组创建Xml对象::
+
+    $data = [
+        'post' => [
+            'id' => 1,
+            'title' => 'Best post',
+            'body' => ' ... '
+        ]
+    ];
+    $xml = Xml::build($data);
+
+如果你的输入参数是无效的，Xml类会抛出一个异常::
+
+    $xmlString = 'What is XML?':
+    try {::
+        $xmlObject = Xml::build($xmlString); // 这儿会抛出一个异常
+    } catch (\Cake\Utility\Exception\XmlException $e) {
+        throw new InternalErrorException();
+    }
+
+.. 提示::
```

Comment:
@panxp yes it is 3.x doc.
I'm just telling you that if you have sphinx tag like ``.. note::`` or ``.. warning:`` etc, it shul not be translated as they are specific markup for sphinx. I sugest that you read the [doc contribution manual page](http://book.cakephp.org/3.0/en/contributing/documentation.html) if you didn't already. Thanks for helpingand contributing anyway. If you get to a point where you don't understand/know how to fix, ping me and I'll gladly help you.